### DISCUSSION TITLE
How to automate publishing to AppStore when app is already approved?

### DISCUSSION THREAD

**Discussion by kambala-decapitator** [fastlane/fastlane#16994] [Q&A] [answered]
Is it possible to publish app to AppStore when it's in Pending Developer Release state? Can't find anything like that in fastlane / spaceship / ASC API, but maybe I'm simply looking in a wrong place.

**Comment by iliao58** [comment]
Spaceship API currently doesn't seem to support although I have already open a discussion. AppStore ConnectAPI now supports this feature though, https://developer.apple.com/documentation/appstoreconnectapi/modify_an_app_store_version_phased_release . It's just not yet included in Spaceship.

**Comment by kambala-decapitator** [comment]
seems it's actually already available, didn't notice: https://github.com/fastlane/fastlane/pull/16765. Will test soon.

**Comment by kambala-decapitator** [comment, marked as answer]
found out that this works: https://github.com/fastlane/fastlane/issues/17140#issuecomment-686493328
